components/automate-cli/pkg/testfiles/aws.auto.tfvars

```diff
@@ -0,0 +1,160 @@
+# AWS
+################################################################################
+aws_profile = "default"
+aws_region  = "us-west-1"
+aws_vpc_id  = "vpc-6fafdtj09"
+aws_cidr_block_addr  = "172.31.0.0"
+private_custom_subnets = []
+public_custom_subnets  = []
+aws_tags    = {
+        X-Contact = ""
+	X-Dept = ""
+	X-Project = ""
+	X-Customer = ""
+	X-Production = ""
+}
+
+aws_ssh_key_pair_name = "abc"
+
+
+ backup_config_s3 = "true"
+
+aws_s3_bucketName = "bucket"
+
+setup_managed_services = false
+managed_opensearch_domain_name = " "
+managed_opensearch_domain_url = " "
+managed_opensearch_username = " "
+managed_opensearch_user_password = " "
+managed_opensearch_certificate = " "
+aws_os_snapshot_role_arn = " "
+os_snapshot_user_access_key_id = " "
+os_snapshot_user_access_key_secret = " "
+managed_rds_instance_url = " "
+managed_rds_superuser_username = " "
+managed_rds_superuser_password = " "
+managed_rds_dbuser_username = " "
+managed_rds_dbuser_password = " "
+managed_rds_certificate = " "
+aws_ami_id = "ami-061534684251"
+lb_access_logs = "false"
+automate_lb_certificate_arn = "arn:aws:acm:us-west-1:11275563:certificate/1a0995"
```

Comment:
confirm if we can allow adding cert-arn in the public repo

Reply:
Its not a valid arn